HISTORY.rst

```diff
@@ -9,7 +9,7 @@ Unreleased
 
 * New lint: Restrict what metric can go into the 'baseline' ping
 
-* New lint: Warn for slight misspellings in ping names
```

Comment:
I think maybe this was a merge conflict resolved incorrectly.  I don't think we want to remove this line (just add the new one below).

Reply:
Hehe, oops. Fixed :)